### DISCUSSION TITLE
Al Dente 2.2 settings persistent in Boot Camp?

### DISCUSSION THREAD

**Discussion by SimonMetcalf** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#286] [Q&A] [answered]
In previous versions of Al Dente, I found the setting "Max Battery Charge" to persist when using Boot Camp.  In the latest version, however, it does not (or isn't reliable) meaning my battery gets charged to capacity whenever I use Windows.  I use Windows frequently to play games, making Al Dente practically ineffective for my needs.

Has this functionality been moved to the Pro version, or is it buggy and not working in the current release?

**Comment by MatthiasKerbl** [comment, marked as answer]
Hi @SimonMetcalf,

we currently use two technical solutions to inhibit charging. The "normal" one does not work in Bootcamp. However, it works much more accurately and even works on M1 Macs. The other one works in Bootcamp but the final charge value can vary by up to 5%. To activate the second method you have to enable "Use Classic SMC Key (Intel)".
